minimal_scheduling/README.md

```diff
@@ -0,0 +1,140 @@
+# minimal_scheduling
+
+These examples show how to configure the thread scheduling policy and priority using pthread APIs.
```

Comment:
```suggestion
These examples show how to configure the thread scheduling policy and priority using [pthread APIs](https://en.wikipedia.org/wiki/Pthreads).
```
I propose to add a link to some good pthread documentation webpage. Probably you know a better one. You might also consider [https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/pthread.h.html](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/pthread.h.html), but it lists the functions only without any description.

Reply:
What about this one? https://linux.die.net/man/7/pthreads